src/target.jl

```diff
@@ -3,6 +3,15 @@
 
 #using Distributions
 
+struct TESSTarget
+  sys::Vector{PlanetarySystemAbstract}
+  snr::Array{Float64,1} # the SNR of observations in each sector?
+  contam::Float64
+  data_span::Float64
+  duty_cycle::Float64
+  # QUERY: do I need a window_function_id?
```

Comment:
Does your detection efficiency depend on a window function?  If not, then no.   If so, then it depends on how you implement it.  If you want to be able to read in data from one on a finite list of tables, then this is a good way to do that.